stylus/ui-app/forms.styl

```diff
@@ -11,9 +11,11 @@ checkbox-size = 20px
 
 $form
     .coz-form
-        position relative
         margin   1em 0 .5em
 
+    .coz-form
+    .coz-form-group
```

Comment:
It does 😄 
I'll wait to see it in action to make my mind but so far that seems like a good reason to add a new class indeed. 😉